documentation/src/main/asciidoc/user_guide/transactions.adoc

```diff
@@ -244,11 +244,11 @@ Pessimistic transactions are more costly by their nature: each write operation p
 
 Write skews occur when two transactions independently and simultaneously read and write to the same key. The result of a write skew is that both transactions successfully commit updates to the same key but with different values.
 
-The `write-skew` attribute automatically detects and rolls back one of the transactions. However, write skew checks apply only to `REPEATABLE_READ` isolation levels in optimistic transactions.
+In Library Mode, {brandname} automatically performs write skew checks to ensure data consistency for `REPEATABLE_READ` isolation levels in optimistic transactions. This allows {brandname} to detect and roll back one of the transactions.
 
 [NOTE]
 ====
-The `write-skew` attribute is deprecated.
+The `write-skew` configuration attribute is deprecated. In Library Mode, {brandname} sets it automatically otherwise the attribute is ignored and does not take effect.
 ====
```

Comment:
If I look to the docs/schema/jboss-infinispan-core_*.xsd the write-skew attribute is not there for server

Reply:
@wfink How about this? "...otherwise the attribute is not available. If you explicitly set `write-skew` attribute in the Infinispan configuration, it does not take effect."

Reply:
AFAICT the server never supported `write-skew` and always failed to start when it was set.
And it was deprecated in embedded mode in 9.0, so it should be safe to remove in 10.0 so that both modes behave the same way. WDYT @tristantarrant?

Reply:
I just noticed that I was replying to a hidden conversation, so I unresolved it.